src/api.ts

```diff
@@ -469,6 +469,7 @@ export class OpenSeaAPI {
       ...opts,
       headers: {
         ...(apiKey ? { "X-API-KEY": apiKey } : {}),
+        ...{ "X-Requested-With": "Opensea-JS" },
```

Comment:
Why are we creating a nested object and spreading it? Can be a plain key: value in the `headers` object.